lte/gateway/c/core/oai/include/amf_securityDef.h

```diff
@@ -84,6 +86,8 @@ class m5g_auth_vector_t {
 #define AUTH_XRES_SIZE AUTH_RES_SIZE
   uint8_t xres_size;
   uint8_t xres[AUTH_XRES_SIZE];
+  uint8_t ck[AUTH_CK_SIZE];
+  uint8_t ik[AUTH_IK_SIZE];
 };
```

Comment:
`m5g_auth_vector_t` should not have any more ck, ik, kasme or xres. Rather it should have kseaf, xres_star

Reply:
this fix also we will  include in "PR1 - 5G Authentication"